src/foam/nanos/auth/ChangePasswordView.js

```diff
@@ -9,267 +9,170 @@ foam.CLASS({
   name: 'ChangePasswordView',
   extends: 'foam.u2.Controller',
 
-  documentation: `Change Password View: managing two states.
-  State one) ResetPassword:  redirect from email link, with a token to update forgotten password,
-  State two) UpdatePassword: simple change of password by a logged in user.
-  
-  Also manages two view types:
-  View one) Vertically aligned properties (this.horizontal = false),
-  View two) Horizontally aligned properties (this.horizontal = true)`,
+  documentation: 'renders a password change model',
 
   imports: [
-    'auth',
-    'notify',
-    'resetPasswordToken',
     'stack',
     'theme',
     'user'
   ],
 
   requires: [
-    'foam.nanos.auth.User',
-    'foam.u2.detail.SectionView'
+    'foam.u2.detail.SectionView',
+    'foam.nanos.auth.SendPassword',
+    'foam.nanos.auth.UpdatePassword',
+    'foam.nanos.auth.ResetPassword'
   ],
 
   css: `
     ^ {
       background: #ffffff;
       height: 100vh;
     }
-    ^ .centerVertical {
-      max-width: 40vw;
-      margin: 6% auto;
-    }
-    ^ .logoCenterVertical {
-      margin: 0 auto;
-      text-align: center;
-      display: block;
-    }
-    ^ .horizontal {
-      padding: 0 0 1vh 2vh;
-      max-width: 98%;
-    }
-    ^ .logoHorizontal {
-      padding-left: 2vh;
-    }
-    ^ .top-bar {
+    ^top-bar {
       background: /*%LOGOBACKGROUNDCOLOUR%*/ #202341;
       width: 100%;
       height: 8vh;
       border-bottom: solid 1px #e2e2e3;
     }
-    ^ .top-bar img {
+    ^top-bar img {
       height: 4vh;
       padding-top: 2vh;
       display: block;
       margin: 0 auto;
     }
-    ^ .title-top {
-      font-size: 2.5em;
-      padding-top: 2vh;
-      font-weight: bold;
-      text-align: center;
-      margin-bottom: 0
+    ^content {
+      max-width: 30vw;
+      margin: 4vh auto 0 auto;
+    }
+    ^section {
+    }
+    /* title  */
+    ^ ^section h2 {
+      margin-top: 0;
+      margin-bottom: 4vh;
+      font-size: 2.5rem;
     }
-    ^ .subtitle {
-      color: #525455;
-      font-size: 1em;
+    /* subtitle */
+    ^ ^section .subtitle {
```

Comment:
What does ^ ^section mean? Is the first ^ necessary?

Reply:
I wanted to give a higher specificity to this property so that it is not overwritten by other 'subtitle' property defined elsewhere.

Reply:
^ ^section .subtitle would be like 'changePasswordView changePasswordview-section subtitle'.

Reply:
Okay, good technique. Maybe add a comment.